.github/workflows/npm-publish.yaml

```diff
@@ -0,0 +1,36 @@
+name: publish
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  npm-publish:
+    name: npm-publish
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@master
+
+      - name: Set up Node.js
+        uses: actions/setup-node@master
+        with:
+          node-version: 10.0.0
+
+      - name: Change directory to package
+      run: cd package
+
+      - name: Install dependencies
+      run: yarn install
+
+      - name: Build
```

Comment:
Aren't we missing a bump step?

Reply:
Indeed, I'll add it

Reply:
Added 6a7e668706c491e6af0e3dca6b269d66ab759d36